source/server/admin/admin.cc

```diff
@@ -124,7 +124,7 @@ AdminImpl::AdminImpl(const std::string& profile_path, Server::Instance& server,
       runtime_handler_(server), listeners_handler_(server), server_cmd_handler_(server),
       server_info_handler_(server),
       // TODO(jsedgwick) add /runtime_reset endpoint that removes all admin-set values
-      handlers_{
+      handlers_ {
```

Comment:
Looks like a true compilation problem here?

Reply:
I think I finally got it... thanks for the patience :slightly_smiling_face: